core/trino-main/src/test/java/io/trino/server/security/TestResourceSecurity.java

```diff
@@ -295,7 +295,7 @@ public void testPasswordAuthenticatorUserMapping()
             // Normally this would result in an impersonation check to the X-Trino-User, but the password
             // authenticator has a hack to clear X-Trino-User in this case.
             Request request = new Request.Builder()
-                    .url(getLocation(httpServerInfo.getHttpsUri(), "/identity"))
+                    .url(getLocation(httpServerInfo.getHttpsUri(), "/protocol/identity"))
```

Comment:
Why not to call `/ui/api/identity` here?

Reply:
This test tests protocol authentication